python/chronos/src/bigdl/chronos/metric/forecast_metrics.py

```diff
@@ -0,0 +1,77 @@
+#
+# Copyright 2016 The BigDL Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+import torch
+from functools import partial
+from torchmetrics.functional import mean_squared_error, mean_absolute_error,\
+    mean_absolute_percentage_error, r2_score, symmetric_mean_absolute_percentage_error
+
+
+TORCHMETRICS_REGRESSION_MAP = {
+    'mae': mean_absolute_error,
+    'mse': mean_squared_error,
+    'rmse': partial(mean_squared_error, squared=False),
+    'mape': mean_absolute_percentage_error,
+    'smape': symmetric_mean_absolute_percentage_error,
+    'r2': r2_score,
+}
+
+
+class Evaluator(object):
+    """
+    Evaluate metrics for y_true and y_pred.
+    """
+
+    @staticmethod
+    def evaluate(metric, y_true, y_pred, multioutput='raw_values'):
+        """
+        Evaluate a specific metric for y_true and y_pred.
+
+        :param metric: String in ['mae', 'mse', 'rmse', 'r2'
+               , 'mape', 'smape']
+        :param y_true: Array-like of shape = (n_samples, \*).
+               Ground truth (correct) target values.
+        :param y_pred: Array-like of shape = (n_samples, \*).
+               Estimated target values.
+        :param multioutput: String in ['raw_values', 'uniform_average']
```

Comment:
"raw_values" and “uniform_average” is too long and does not accurately describe what it means ("raw_values" actually means "element-wise", and "uniform_average" actually means aggregated by mean (comparing againse element-wise). Actually the hyper parameter "multioutput“ itself is confusing as well. 

My suggestion is: If we have only two options here, we can use a bool parameter, e.g. aggrerate=True or element-wise=True. If we have more than two aggregation option plus an element-wise option, then we can use aggregate='None', 'mean', etc.

Reply:
This setting is inherited from `orca.automl` (which inherited from `sklearn`). I think a string type `aggregate` is much better since we do plan to add more settings for our users' convenience.